browser/src/control/Control.Menubar.js

```diff
@@ -1681,6 +1684,16 @@ L.Control.Menubar = L.Control.extend({
 						} else {
 							$(aItem).text(_('Use Tabbed view'));
 						}
+					} else if (id === 'showslide') {
+						if (!self._map._docLayer.isHiddenSlide(self._map.getCurrentPartNumber()))
+							$(aItem).addClass('disabled');
+						else
+							$(aItem).removeClass('disabled');
+					} else if (id === 'hideslide') {
+						if (self._map._docLayer.isHiddenSlide(self._map.getCurrentPartNumber()))
+							$(aItem).addClass('disabled');
+						else
+							$(aItem).removeClass('disabled');
```

Comment:
Can we please instead of adding/removing disable css class just show/hide the specific menu item? So instead of seeing 2 options (hide slide and show slide) just one of those:

![image](https://github.com/CollaboraOnline/online/assets/65948705/e441f8f8-83fc-47c4-8d9d-914955361e55)

```diff
-                                                       $(aItem).addClass('disabled');
+                                                       $(aItem).hide();
-                                                       $(aItem).removeClass('disabled');
+                                                       $(aItem).show();
```